index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav>
+        <header>
+            <a href="#">
+                <img class="brang-img" src="./images/Brand-logo.png" alt="brang-logo" />
+            </a>
+            <img class="hamburger-menu" src="./images/Icon - Menu.png" alt="hamburger-menu">
+        </header>
+    </nav>
+    <div class="showcase">
+        <div class="hero">
+            <h1>Hey There.
+                I’m Jakaria</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="icons">
```

Comment:
[ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social net works with the icons, using href. kindly fix that 🤓